### ISSUE TITLE
Superset Map visualization deck.gl Scatter plot does not work and not shown

### ISSUE THREAD

**Issue by Bapawin** [apache/superset#30763]
### Bug description

<img width="497" alt="mapbroken" src="https://github.com/user-attachments/assets/7a837315-2212-4955-951b-8db3b7500225">
I try to create chart of Map visualization deck.gl Scatter plot or any map type but it shown only my latitude&longtitude data but the background map disappear

So I try to edit config file 
![image](https://github.com/user-attachments/assets/ca6d9fac-fe72-494e-9721-07fe96bffc3e)
I create token from mapbox website and change to MAPBOX_API_KEY = "mytoken" but still not worked

I create superset in docker.

### Superset version

master / latest-dev

### Python version

3.9

### Node version

I don't know

### Browser

Chrome

### Checklist

- [X] I have searched Superset docs and Slack and didn't find a solution to my problem.
- [X] I have searched the GitHub issue tracker and didn't find a similar bug report.
- [ ] I have checked Superset's logs for errors and if I found a relevant Python stacktrace, I included it here as text in the "additional context" section.

**Comment by rusackas** [comment]
Did you restart Superset after adding the token?

**Comment by zero-stroke** [comment]
Why should an api key even be required to render a basic non-satellite map along with some coordinates?

**Comment by Sahilpatkar** [comment]
Try using a different browser like firefox it worked for me. 
I spent a lot of time trying to make it run on chrome.. later realised that WebGL wasn't enabled on my chrome.

**Comment by zero-stroke** [comment]
> Try using a different browser like firefox it worked for me. I spent a lot of time trying to make it run on chrome.. later realised that WebGL wasn't enabled on my chrome.

Using firefox allows you to load a basic map, without having to use an api key?

**Comment by Sahilpatkar** [comment]
No, i did use the api key..just that it still did not work on chrome. 
I was able to see the maps when i switched to firefox.

**Comment by Bapawin** [comment]
I already fix it now but I have problem about this issue instead as: https://github.com/apache/superset/issues/31746

**Comment by rusackas** [comment]
Just noting that a PR is open to replace Mapbox as the default with OpenStreetMaps:
https://github.com/apache/superset/pull/33603

Please provide any testing/feedback over there.
